agent/app/agent_capability.go

```diff
@@ -438,6 +446,18 @@ func (agent *ecsAgent) appendExecCapabilities(capabilities []*ecs.Attribute) ([]
 	return appendNameOnlyAttribute(capabilities, attributePrefix+capabilityExec), nil
 }
 
+func (agent *ecsAgent) appendServiceConenctCapabilities(capabilities []*ecs.Attribute) ([]*ecs.Attribute, error) {
+	if loaded, _ := agent.appNetLoader.IsLoaded(agent.dockerClient); !loaded {
+		_, err := agent.appNetLoader.LoadImage(agent.ctx, agent.dockerClient)
+		if err != nil {
+			seelog.Warn("ServiceConnect Capability: Failed to load appnet Agent container")
```

Comment:
I would maybe log an ERROR message here and describe the issue that it will cause, and include the err in the error message (if possible also using the agent's structured logging package which it looks like this file doesn't currently use):
```go
import (
	"github.com/aws/amazon-ecs-agent/agent/logger"
	"github.com/aws/amazon-ecs-agent/agent/logger/field"
)

...

logger.Error("ServiceConnect Capability: Failed to load appnet Agent container. This container instance will not be able to support ServiceConnect tasks", logger.Fields{field.Error: err})
return capabilities, nil
```

Reply:
What is the guidance on seelog versus logger? It isn't consistent in the codebase

Reply:
logger is newer and what we prefer because it's structured and easier to read, but we still have some places directly using seelog just because I dont think anyone has had the time to fully migrate everything yet.